src/data/weather.ts

```diff
@@ -77,13 +77,29 @@ const getWindBearingText = (degree: string): string => {
   return degree;
 };
 
-export const getWindBearing = (bearing: string): string => {
+const getWindBearing = (bearing: string): string => {
   if (bearing != null) {
     return getWindBearingText(bearing);
   }
   return "";
 };
 
+export const getWind = (
+  hass: HomeAssistant,
+  speed: string,
+  bearing: string
+): string => {
+  if (bearing != null) {
```

Comment:
```suggestion
  if (bearing !== null) {
```

Found one little thing here which I think may have been in the old code. Bram will review this as well and merge if good :)

Reply:
looks good to me.